src/Conference/ConferenceDetails.jsx

```diff
@@ -18,7 +19,13 @@ export const StyledDescription = styled.p`
   padding: 0 100px 50px 100px;
 `;
 
-const ConferenceDetails = ({ conference, id, isLoading, error }) => {
+const ConferenceDetails = ({
+  conference,
+  id,
+  isLoading,
+  error,
+  allCookies = {}
+}) => {
   const history = useHistory();
```

Comment:
might be nice to make a variable here. `const sessionToken = allCookies.sessiontoken` as you use it in a couple places throughout the file

As the default is {}, youll have to do a check `allCookies ? allCookies.sessiontoken : null`

Reply:
Likewise in other files as well